terraform/monitoring/terraform.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+
+  backend "s3" {
+    key     = "monitoring/terraform.tfstate"
+    region  = "eu-west-2"
+    encrypt = "true"
+  }
+}
+
+provider cloudfoundry {
+  api_url           = var.paas_api_url
+  password          = var.paas_password
+  sso_passcode      = ""
```

Comment:
Uhh, do we need to set this to empty string?

Reply:
🤷 let's find out